### ISSUE TITLE
[Bug] Incorrect user has reject the transaction error

### ISSUE THREAD

**Issue by 0xean** [shapeshift/web#8035]
![image](https://github.com/user-attachments/assets/e70d967d-58ee-44e5-af6f-1f1d734002a1)

We have a user that reach out to support with the following issues attempting to send USDT on polygon from an account with no POL (native asset). This is odd for a few reasons because on desktop we correctly block the action like this (created by spoofing their account)

![image](https://github.com/user-attachments/assets/044676e8-2a57-463e-b0bc-381b7564818a)

A few things to note:

1. The from and to addresses are both valid (please ping me and I can provide them as needed )
2. They were able to recreate this issue repeatedly

**Comment by 0xean** [comment]
Another desktop user has reported a similar issue, they were attempting to send BTC, from and to are both valid addresses.

**Comment by NeOMakinG** [comment]
The first case with POL is pretty easy to understand:
![Image](https://github.com/user-attachments/assets/ee015ec0-d00b-43fc-8f00-18137847360d)

As seen in the screenshot, the gasFee estimation can be variating a lot between the input and the moment we sent the TX

If your balance is pretty close to the gas estimation, but under, you have the 🟢 light to send the TX, but if you wait a few seconds, then click on the button, the gas can be over the current balance, resulting to a TX revert

If you are using desktop, you might be using MM or another wallet, so the wallet itself will say it properly and this is understandable

If you are using mobile, you are using the native wallet which doesn't show anything about gas estimation 

Let's discuss this on wednesday: 
- Poll fee estimation on the send modal?
- Reestimate fees right before sending?

I'm checking if I can update the catch message to inform users of what is happening

**Comment by 0xean** [comment]
> If your balance is pretty close to the gas estimation, but under, you have the 🟢 light to send the TX, but if you wait a few seconds, then click on the button, the gas can be over the current balance, resulting to a TX revert

I don't think this is correct. The user had 0 POL and was able to get passed the disabled send button on mobile.  There should never be a case when gas estimation would explain this behavior.

**Comment by NeOMakinG** [comment]
> > If your balance is pretty close to the gas estimation, but under, you have the 🟢 light to send the TX, but if you wait a few seconds, then click on the button, the gas can be over the current balance, resulting to a TX revert
> 
> I don't think this is correct. The user had 0 POL and was able to get past the disabled send button on mobile. There should never be a case when gas estimation would explain this behavior.

I just reproduced the same behavior, I'll fix it!

In the meantime I pushed a first quickwin to have a more verbose error message for the case I written before
